C#Backend/InpatientTherapySchedulingProgramTests/ControllerTests/HoursWorkedControllerTests.cs

```diff
@@ -0,0 +1,206 @@
+﻿using InpatientTherapySchedulingProgram.Controllers;
+using InpatientTherapySchedulingProgram.Models;
+using InpatientTherapySchedulingProgramTests.Fakes;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Moq;
+using InpatientTherapySchedulingProgram.Services.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+using InpatientTherapySchedulingProgram.Exceptions.HoursWorkedExceptions;
+
+namespace InpatientTherapySchedulingProgramTests.ControllerTests
+{
+    [TestClass]
+    public class HoursWorkedControllerTests
+    {
+        private static List<HoursWorked> _testHoursWorked;
+        private Mock<IHoursWorkedService> _fakeService;
```

Comment:
Small semantics and not a big deal but renaming from _fakeService to _fakeHoursWorkedService may make it sound more descriptive.